<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MorphOpsDialog</class>
 <widget class="QDialog" name="MorphOpsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Morphological Operations Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>351</width>
     <height>153</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="dilationRadioButton">
      <property name="text">
       <string>Dilation</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="erosionRadioButton">
      <property name="text">
       <string>Erosion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="openingRadioButton">
      <property name="text">
       <string>Opening</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="closingRadioButton">
      <property name="text">
       <string>Closing</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="thinningRadioButton">
      <property name="text">
       <string>Hit-Or-Miss Thinning</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="thickeningRadioButton">
      <property name="text">
       <string>Hit-Or-Miss Thickening</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>WARNING: This will binarize the image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MorphOpsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MorphOpsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
